py/parse_and_notify.py

```diff
@@ -145,53 +149,46 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
 @dataclass(frozen=True)
 class TopicctlOutput:
     dry_run: bool
-    topics: Sequence[Topic]
+    topic: Topic
 
     @classmethod
     def build(cls, raw_content: str) -> TopicctlOutput:
         parsed = json.loads(raw_content)
         return TopicctlOutput(
             dry_run=parsed["dryRun"],
-            topics=[
-                *[
-                    NewTopic.build(content)
-                    for content in parsed["newTopics"] or []
-                ],
-                *[
-                    UpdatedTopic.build(content)
-                    for content in parsed["updatedTopics"] or []
-                ],
-            ],
+            topic=NewTopic.build(parsed["newTopic"])
+            if parsed["newTopic"]
+            else UpdatedTopic.build(parsed["updatedTopic"]),
```

Comment:
Please add an assertion in case there is an invalid value. So the error message will be specific rather than seeing a failure when you trying to use a data structure that is not compatible with. `updatedTopic`.

```
if parsed["newTopic"]:
  ...
elif parsed["updatedTopic"]:
  ....
else:
   raise ValueError.
```